src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfEntries = sourceString.split(';');
+  const arrayOfKeysAndValues = arrayOfEntries.map((entry) => entry.split(':'));
+  const trimmedArray = arrayOfKeysAndValues.map(function (item) {
+    if (item.length === 2) {
+      return [item[0].trim(), item[1].trim()];
+    }
+
+    return item;
+  });
+  const filtredArray = trimmedArray.filter((entry) => entry.length > 1);
```

Comment:
There is a typo in the variable name `filtredArray`. It should be `filteredArray` to correctly reflect the operation being performed.